deps/hiredis/adapters/redismoduleapi.h

```diff
@@ -1,7 +1,7 @@
 #ifndef __HIREDIS_REDISMODULEAPI_H__
 #define __HIREDIS_REDISMODULEAPI_H__
 
-#include "redismodule.h"
+#include "valkeymodule.h"
```

Comment:
Please don't change this. Hiredis is a dependency copied from another repository, so we want to fix it upstream, maybe fork hiredis. This file isn't used by valkey btw.

```suggestion
#include "redismodule.h"
```